license-header-check/action.yml

```diff
@@ -36,10 +41,14 @@ runs:
         CURRENT_YEAR=$(date +%Y)
         LICENSE_PATTERN="Copyright \(c\) .*?${CURRENT_YEAR},? NVIDIA CORPORATION."
 
-        # Skip for merge to main PR
-        if [[ "${{ github.event.pull_request.base.ref }}" == "main" ]]; then
-          echo "Skip this check for Merge to main PR."
-          exit 0
+        # Skip check for main branch when check_main == false
+        if [[ "${{ inputs.check_main }}" == "false" ]]; then
+          if [[ "${{ github.event.pull_request.base.ref }}" == "main" ]]; then
+            echo "Skip this check for Merge to main PR."
+            exit 0
+          fi
+        else
```

Comment:
nit: seems no need to echo this line for `non-main` case. could simply remove the else part

Reply:
removed